README.md

```diff
@@ -133,6 +148,17 @@ supply a custom `RateLimitKeyGenerator` implementation adding further qualifiers
           }
       };
   }
+  
+  @Bean
+  public UserIdGetter userIdGetter() {
```

Comment:
should be named UserIDGenerator and DefaultUserIDGenerator

Reply:
ok